cost-analyzer/templates/NOTES.txt

```diff
@@ -1,9 +1,35 @@
 
 
 --------------------------------------------------
+{{- $node := (lookup "v1" "Node" "" "") }}
+{{- $isEKS := (regexMatch ".*eks.*" (.Capabilities.KubeVersion | quote) )}}
+{{- $isGT22 := (semverCompare ">=1.23-0" .Capabilities.KubeVersion.GitVersion) }}
+{{- $PVNotExists := (empty (lookup "v1" "PersistentVolume" "" "")) }}
+{{- $EBSCSINotExists := (empty (lookup "apps/v1" "Deployment" "kube-system" "ebs-csi-controller")) }}
```

Comment:
would you ever not have namespace access here? if you don't have access, what happens?

Reply:
IMO you should get the warning.

But I think after a quick test this actually will block install. Let me see if it's quick to add a try-catch of some sort here.

Reply:
Yep, it will block install:
```
The lookup function uses Helm's existing Kubernetes connection configuration to query Kubernetes. If any error is returned when interacting with calling the API server (for example due to lack of permission to access a resource), helm's template processing will fail.
```